src/test/java/uk/gov/laa/ccms/caab/opa/context/exception/ConnectorSecurityContextExceptionTest.java

```diff
@@ -0,0 +1,43 @@
+package uk.gov.laa.ccms.caab.opa.context.exception;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Could be expanded to single imports

Reply:
I will raise a change for that

Reply:
https://github.com/ministryofjustice/laa-ccms-caab-helm-charts/pull/10

Reply:
Amended